private_gpt/settings/settings.py

```diff
@@ -508,6 +508,28 @@ class QdrantSettings(BaseModel):
     )
 
 
+class MilvusSettings(BaseModel):
+    uri: str = Field(
+        "local_data/private_gpt/milvus_local.db",
```

Comment:
Probably, I don't explain me well. The idea is having a folder inside `local_data/private_gpt` with each provider. In this case, it would be: `local_data/private_gpt/milvus/milvus_loca.db`

Reply:
Oh i see, thank you for the declarations.